scripts/test_notebooks.py

```diff
@@ -99,8 +99,12 @@ def notebook_find_replace(notebook, find_sent_regex, find_str_regex, replace_str
 
 ## Env vars
 CLIENT_INSTANTIATION_SENT_REGEX = '"client.*Client(.*)"'
+
 TEST_PROJECT = os.getenv("TEST_PROJECT")
 TEST_API_KEY = os.getenv("TEST_API_KEY")
+if TEST_PROJECT == None or TEST_API_KEY == None:
```

Comment:
If you want breaking behavior if env variable isn't present, you can just use this: 

```
os.environ["TEST_PROJECT"]
os.environ["TEST_API_KEY"]
```

Reply:
thank you ~ way more elegant.